buildSrc/src/main/groovy/airbyte-integration-test-java.gradle

```diff
@@ -25,7 +25,11 @@ class AirbyteIntegrationTestJavaPlugin implements Plugin<Project> {
             testClassesDirs += project.sourceSets.integrationTestJava.output.classesDirs
             classpath += project.sourceSets.integrationTestJava.runtimeClasspath
 
-            useJUnitPlatform()
+            useJUnitPlatform {
+                // todo (cgardens) - figure out how to de-dupe this exclusion with the one in build.gradle.
```

Comment:
One solution would be to make an include list instead and properly annotate the integration test.